content/strategy-goals/strategy/code-graph/index.md

```diff
@@ -1,30 +1,24 @@
+Code Graph comprises teams working on [Search](./search/index.md), [Code Insights](./code-insights/index.md), [Code Intelligence](./code-intelligence/index.md), and [Batch Changes](./batch-changes/index.md); this page describes the mission and strategy that ties them together over the next year or so, with each linked page providing a more detailed strategy for that area. There is also a whole-company [Sourcegraph strategy](../index.md) and [Product & Engineering strategy](../../../departments/product-engineering/strategy-goals/index.md) to which all of this is aligned. Our [roadmap tracker](https://github.com/orgs/sourcegraph/projects/214/views/34) (internal only) shows what we're delivering to make this strategy a reality.
 
-If you're new to our strategy pages, this video walkthrough will help orient you.
+If you're new to our strategy pages, this video walkthrough (last updated Feb 2022) will help orient you.
 
 <div style="position: relative; padding-bottom: 56.25%; height: 0;"><iframe src="https://www.loom.com/embed/71b675b67c464b0db7670a7f5025cbbe" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen style="position: absolute; top: 0; left: 0; width: 100%; height: 100%;"></iframe></div>
 
-## Themes
+## Mission
 
-Our mission is to **make discovering, learning, and editing code easier, no matter how complex it is, where it's stored, or how experienced you are.** We will achieve this through several important themes:
+Our mission is to **make discovering, learning, and editing code easier, no matter how complex it is, where it's stored, or how experienced you are.**
 
-### Make the power of our features easier to find and use
+## Current theme(s)
 
-Getting the most out of our product can take time as you get up to speed with what's possible. Making it easier to learn will help teams find the value more quickly than today. Our task is to find a way to make interacting with multiple languages and repositories easy to understand.
+Apart from simply delivering great features that make our mission possible, across Code Graph we're particularly focused at the moment on improving the **discoverability and ease-of-use** around our [use cases](../index.md#use-cases). Focusing on these ensures we're prioritizing areas where people get real value out of the product, reducing time-to-value reduces the chance of people giving up on our features because they are too complicated or are not intuitive, and building cross-team capabilities helps us build better features that are more than the sum of their parts (as opposed to shipping our org chart).
```

Comment:
```suggestion
Apart from the work each of our teams do for our mission, across Code Graph we're particularly focused now on improving the **discoverability and ease-of-use** around our [use cases](../index.md#use-cases). Focusing on these ensures we're prioritizing areas where people get real value out of the product, reducing time-to-value reduces the chance of people giving up on our features because they are too complicated/not intuitive, and building cross-team capabilities helps us deliver features that are more than the sum of their parts (as opposed to shipping our org chart).
```